task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -33,6 +35,7 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
+        echo "$(params.BASE_IMAGES_DIGESTS)"
```

Comment:
What happens if an image from a non-Red Hat registry is used? We can catch such things in the Enterprise Contract, but I don't think we want the build pipeline to fail.

Reply:
it's covered in lines [60-67](https://github.com/redhat-appstudio/build-definitions/blob/6cf78ece535f3e07319405aec089eea58731e706/task/deprecated-image-check/0.1/deprecated-image-check.yaml#L57) in this PR